src/viewport/observability.py

```
@@ -0,0 +1,303 @@
+            SQLAlchemyInstrumentor().instrument(engine=sync_engine)
+        SQLAlchemyInstrumentor().instrument(engine=get_sync_engine())
+        _DB_INSTRUMENTED = True
+    except Exception:
+        logger.debug("OpenTelemetry SQLAlchemy instrumentation unavailable", exc_info=True)
+
+
+def _endpoint_is_insecure(endpoint: str) -> bool:
+    return endpoint.startswith("http://") or endpoint.startswith("localhost") or endpoint.startswith("127.0.0.1")
+
+
+def _sanitize_readable_span(span: "ReadableSpan") -> None:
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)